src/components/Modeler.vue

```diff
@@ -215,6 +215,7 @@ export default {
       if (this.tooltipTarget) {
         return this.tooltipTarget.$el.data('title');
       }
+      return undefined;
```

Comment:
After updates to eslint this was flagged as a compilation error since computed properties need to return a value.